.github/workflows/test-php.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: test php
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: '["self-hosted", "some-runner"]'
+        required: true
+        type: string
+      php-cs-fixer-cmd:
+        description: optional args for php-cs-fixer
+        required: false
+        type: string
+        default: null
+      php-stan-cmd:
+        description: path(s) for php stan to check
+        required: false
+        type: string
+        default: analyse /src
+
+jobs:
+  test-php:
+    name: php
+    runs-on: ${{fromJSON(inputs.runner)}}
+    steps:
+      # https://github.com/marketplace/actions/checkout
+      - name: checkout
+        uses: actions/checkout@v3.5.0
+
+      # https://github.com/marketplace/actions/php-lint
+      - name: php-lint
```

Comment:
Will check in next php projects if it works as expected as we dont use `vendor/bin/phplint` on composer projects. But looks

Reply:
@jazzlyn  this php-lint will make issues with our spunq libs. We need to make it optional